scripts/sanity_chk/sanitylib.py

```diff
@@ -2058,6 +2058,7 @@ def process(self, message):
                 self.instance.reason = "Build Failure"
                 pipeline.put({"op": "report", "test": self.instance})
             else:
+                self.instance.status = "passed"
```

Comment:
I wonder if it won't be better if we we only fill the status with 'passed' for instances with build_only == True 
e.g.: 
```
            else:
                if self.instance.build_only:
                    self.instance.status = "passed"
```